airflow/utils/operator_helpers.py

```diff
@@ -77,6 +87,21 @@ def context_to_airflow_vars(context: Mapping[str, Any], in_env_var_format: bool
         (dag_run, 'run_id', 'AIRFLOW_CONTEXT_DAG_RUN_ID'),
     ]
 
+    context_params = settings.get_airflow_context_vars(context)
+    for key, value in context_params.items():
+        if not isinstance(key, str):
+            raise TypeError("key must be string")
+        if not isinstance(value, str):
+            raise TypeError("value must be string")
```

Comment:
Can we show the value of `key` and `value` in the exception message? This would help the user debug the problem. Something like `f"key must be string, not {key!r}"` would be enough (and the value variant should show both `key` and `value`).

Reply:
i think this should be clear enough. but it wouldn't hurt to be more explicit.